app/_gateway_entities/consumer.md

```diff
@@ -4,60 +4,60 @@ content_type: reference
 
 related_resources:
-  - text: Authentication reference
-    url: https://docs.konghq.com/gateway/latest/kong-plugins/authentication/reference/
-  - text: Consumer groups API reference
+  - text: Authentication in {{site.base_gateway}}
+    url: /authentication/
+  - text: Consumer Groups API reference
     url: /api/gateway/admin-ee/#/operations/get-consumer_groups
-  - text: Plugins that can be enabled on consumers
-    url: https://docs.konghq.com/hub/plugins/compatibility/#scopes
+  - text: Plugins that can be enabled on Consumers
+    url: /plugins/scopes/
 
 faqs:
   - q: What are credentials, and why do I need them?
     a: |
-      Credentials are necessary to authenticate consumers via various authentication mechanisms.
+      Credentials are necessary to authenticate Consumers via various authentication mechanisms.
       The credential type depends on which authentication plugin you want to use.
       <br><br>
-      For example, a Key Authentication plugin requires an API key, and a Basic Auth plugin requires a username and password pair.
+      For example, a Key Authentication plugin requires an API key, and a Basic Authentication plugin requires a username and password pair.
 
-  - q: What is the difference between consumers and applications?
+  - q: What is the difference between Consumers and Applications?
     a: |
-      Applications provide developers the ability to get access to APIs managed by {{site.base_gateway}} or Konnect
-      with no interaction from the Kong admin team to generate credentials required.
+      Applications provide developers the ability to get access to APIs managed by {{site.base_gateway}} or {{site.konnect_short_name}}
+      with no interaction from the Kong admin team to generate the required credentials.
       <br><br>
-      With consumers, the Kong team creates consumers, generates credentials and needs to share them with the developers that need access to the APIs.
-      You can think as applications as a type of consumer in Kong that allows developers to automatically obtain credentials for and subscribe to the required APIs.
+      With Consumers, the Kong team creates Consumers, generates credentials, and shares them with the developers that need access to the APIs.
+      You can think of Applications as a type of Consumer in Kong that allows developers to automatically obtain credentials for, and subscribe to the required APIs.
 
-  - q: What is the difference between consumers and developers?
+  - q: What is the difference between Consumers and Developers?
     a: |
-      Developers are real users of the Dev Portal, whereas consumers are abstractions.
+      Developers are real users of the Dev Portal, whereas Consumers are abstractions.
 
-  - q: What is the difference between consumers and RBAC users?
+  - q: What is the difference between Consumers and RBAC Users?
```

Comment:
Yeah, I think we have to leave them like this for consistency. Otherwise we'll start arbitrarily drawing the line between what is "core" and what isn't, which isn't actually a set definition.